monitoring/dashboard.py

```diff
@@ -525,6 +559,167 @@ def top10_errors_by_bucket(title, code):
+        average_quota_latency_target(
+            title="Data write (bucket, success)",
+            type='"bucket"', code='~"2.."'),
+        average_quota_latency_target(
+            title="Data write (bucket, exceeded)",
+            type='"bucket"', code='"429"'),
+        average_quota_latency_target(
+            title="Data write (account, success)",
+            type='"account"', code='~"2.."'),
+        average_quota_latency_target(
+            title="Data write (account, exceeded)",
+            type='"account"', code='"429"'),
+        average_quota_latency_target(
+            title="Data write (account & bucket, success)",
+            type='"bucketAccount"',
+            code='~"2.."'),
+        average_quota_latency_target(
+            title="Data write (account & bucket, exceeded)",
+            type='"bucketAccount"',
+            code='"429"'),
+        average_quota_latency_target(
+            title="Data deletion", type='"delete"'),
+    ],
+)
+
+
+averageMetricsRetrievalLatencies = TimeSeries(
+    title="Average utilization metrics retrieval latencies",
+    dataSource="${DS_PROMETHEUS}",
+    lineInterpolation="smooth",
+    spanNulls=3*60*1000,
+    unit=UNITS.SECONDS,
+    targets=[
+        average_quota_retrieval_latency(title="Bucket (success)",
+                                        _class='"bucket"', code='~"2.."'),
+        average_quota_retrieval_latency(title="Account (success)",
+                                        _class='"account"', code='~"2.."'),
+        average_quota_retrieval_latency(title="Bucket (error)",
+                                        _class='"bucket"', code='~"4..|5.."'),
+        average_quota_retrieval_latency(title="Account (error)",
+                                        _class='"account"', code='~"4..|5.."'),
+    ],
+)
+
+
+bucketQuotaCounter = Stat(
+    title="Buckets with quota",
+    description=(
+        "Number of S3 buckets with quota enabled in the cluster.\n"
+        "This value is computed asynchronously, and update "
+        "may be delayed up to 1h."
+    ),
+    dataSource="${DS_PROMETHEUS}",
+    colorMode="value",
+    format=UNITS.SHORT,
+    noValue="0",
+    reduceCalc="lastNotNull",
+    targets=[Target(
+        expr='sum(s3_cloudserver_buckets_quota_count{namespace="${namespace}"})',  # noqa: E501
+    )],
+    thresholds=[
+        Threshold("#808080", 0, 0.0),
+        Threshold("blue", 1, 0.0),
+    ],
+)
+
+
+accountQuotaCounter = Stat(
+    title="Accounts with quota",
+    description=(
+        "Number of accounts with quota enabled in the cluster.\n"
+        "This value is computed asynchronously, and update "
+        "may be delayed up to 1h."
+    ),
+    dataSource="${DS_PROMETHEUS}",
+    colorMode="value",
+    format=UNITS.SHORT,
+    noValue="0",
+    reduceCalc="lastNotNull",
+    targets=[Target(
+        expr='sum(s3_cloudserver_account_quota_count{namespace="${namespace}"})',  # noqa: E501
```

Comment:
`max()` may be more appropriate than `sum`, it case we have multiple series at the same time (e.g. pod update maybe?)

Reply:
Indeed, see https://github.com/scality/cloudserver/pull/5556/files#diff-9534a9dab1150e97d07f2e6a3541659b7d37c0aa2fb7db06e95f25fc3864078fR696